packages/java-edition/src/mcfunction/tree/patch-validator.ts

```diff
@@ -0,0 +1,37 @@
+import type * as core from '@spyglassmc/core'
```

Comment:
The dash in this file name seems a bit odd, no? I would expect `patchValidator.ts`

Reply:
Or patch_validator